cloud

```diff
@@ -1249,6 +1250,7 @@ function cloud_function_withdraw() {
 function cloud_functions_env() {
     params="$(getopt_vars format \
                           log_level \
+                          optimize_level \
```

Comment:
Could you call it just `optimize` everywhere? It's shorter, and uniformity helps understanding. Besides, this doesn't match the function documentation above.

Reply:
Okay